Ginger/GingerCore/Actions/Common/ActUIElement.cs

```diff
@@ -436,6 +436,10 @@ public enum eElementAction
             IsDisabled,
             [EnumValueDescription("Switch")]
             Switch,
+            [EnumValueDescription("XYDoubleClick")]
```

Comment:
Description change it to Double Click using XY and Send Keys Using XY

And Enum value change it to DoubleClickXY and SendKeysXY